tmp/build/Dockerfile

```diff
@@ -1,6 +1,24 @@
-FROM alpine:3.6
+#FROM alpine:3.6
```

Comment:
we should be basically ignoring this file at this point.  The one that matters is the Dockerfile in the root of the repo.